app/src/main/java/com/beemdevelopment/aegis/ui/views/EntryListView.java

```diff
@@ -191,6 +208,14 @@ public void onDestroyView() {
         super.onDestroyView();
     }
 
+    public void onRefreshStop() {
+        _refresher.stop();
+    }
+
+    public void onRefreshStart() {
```

Comment:
Since we're now unconditionally starting the refresher from MainActivity, I think we need to check whether ``_adapter.getMostFrequentPeriod()`` returns ``-1`` here and if so not start the refresher. Otherwise, this causes a tight loop in the refresher in case there are no entries in the vault yet. This is also the reason why the instrumented tests are failing.